src/kits/kitsController.ts

```diff
@@ -276,6 +280,32 @@ export class KitsController {
             if (chosen_kit.kit.name === SpecialKits.ScanForKits) {
                 await KitsController.scanForKits(await this.project.getCMakePathofProject());
                 return false;
+            } else if (chosen_kit.kit.name === SpecialKits.ScanSpecificDir) {
+                const dir = await vscode.window.showOpenDialog({
+                    canSelectFiles: false,
+                    canSelectFolders: true,
+                    canSelectMany: false,
+                    openLabel: localize('select.folder', 'Select Folder')
```

Comment:
Also, I think it may be a good idea to ensure somewhere that it will also search recursively within the folder. Either here or in the popup that asks to either scan for kits or choose a specific folder.

Reply:
Updated other strings.